bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfileTest.java

```diff
@@ -145,7 +145,7 @@ private static Stream<Arguments> provideTestCasesForApiDetermination() {
     @ParameterizedTest
     @MethodSource("provideTestCasesForBuildBluServiceName")
     void buildBluServiceName(String name, String mac, String expectedServiceName) {
-        String actualServiceName = ShellyDeviceProfile.buildBluServiceName(name, mac);
+        String actualServiceName = getBluServiceName(name, mac);
```

Comment:
This test should now be moved to `ShellyDevicesTest` and renamed to `getBluServiceName`.

Reply:
done